src/store/modules/activity_daily.ts

```diff
@@ -35,17 +35,26 @@ interface QueryOptions {
 const _state = {
   top_apps: [],
   top_titles: [],
+  app_chunks: [],
+
   top_domains: [],
   top_urls: [],
+  web_chunks: [],
```

Comment:
`app_chunks` and `web_chunks` aren't used anymore (because they make the query vastly more expensive).

Reply:
Done